homeassistant/components/zwave/climate.py

```diff
@@ -66,6 +73,24 @@
     "auto changeover": HVAC_MODE_HEAT_COOL,
 }
 
+MODE_SETPOINT_MAPPINGS = {
+    "off": (),
+    "heat": ("setpoint_heating",),
+    "cool": ("setpoint_cooling",),
+    "auto": ("setpoint_heating", "setpoint_cooling"),
+    "aux heat": ("setpoint_heating",),
+    "furnace": ("setpoint_furnace",),
+    "dry air": ("setpoint_dry_air",),
+    "moist air": ("setpoint_moist_air",),
+    "auto changeover": ("setpoint_auto_changeover",),
+    "heat econ": ("setpoint_eco_heating",),
```

Comment:
The eco mode is called "Heat Eco" in case of Eurotronic Spirit-Z thermostats. I'm not sure if this makes a difference.

Reply:
Thanks for noticing that.
The  labels I used are from https://github.com/OpenZWave/open-zwave/blob/85ca5769a57d13f328e65b415d95e95647994366/cpp/src/command_classes/ThermostatMode.cpp#L56-L94 .
But it turns out open-zwave let's you override the labels in xml as well.
In your case here: https://github.com/OpenZWave/open-zwave/blob/85ca5769a57d13f328e65b415d95e95647994366/config/eurotronic/eur_spiritz.xml#L71
So indexes are the same but labels are different :(
I just ran grep across all XML configs and found various confusing labels for same indexes e.g.
`Energy Heat = Energy Saving = Heat Eco = Heat Econ` and so on.

I will add the aliases I've found.
Sometime later it makes sense to switch to indexes instead of aliases since they seem to be more stable